locales/sv/election.json

```diff
@@ -43,5 +43,7 @@
   "party": "Parti",
   "privacyNote": "Genom att klicka på »Starta VoteSwiper« godkänner du vår <0>integritetspolicy</0>.",
   "donationTitle": "Support this project",
-  "donationText": "<0>The VoteSwiper is a project made by <1>citizens for citizens</1>. Today we reach several million people with - and that does not only require a lot of time, which we are happy to invest, but also money: The hosting alone for larger elections quickly costs <1>several thousand euros</1>. We are therefore dependent on your support to be able to continue offering VoteSwiper for you in the future.</0><0>Every amount helps.</0>"
+  "donationText": "<0>The VoteSwiper is a project made by <1>citizens for citizens</1>. Today we reach several million people with - and that does not only require a lot of time, which we are happy to invest, but also money: The hosting alone for larger elections quickly costs <1>several thousand euros</1>. We are therefore dependent on your support to be able to continue offering VoteSwiper for you in the future.</0><0>Every amount helps.</0>",
+  "skippedQuestionsTitle": "Du har hoppat över alla frågor Du har hoppat över alla frågor.",
```

Comment:
I believe you pasted the text twice here

Reply:
woops totaly overlooked that one haha. thanks for pointig it out, i fixed it